### ISSUE TITLE
 should we change project validate --project to validate --project in the cli?

### ISSUE THREAD

**Issue by samuelstroschein** [inlang/monorepo#1777]
## Problem 

#1762 introduces the inlang CLI v2 because `--project` flag is now always required because no assumption of naming or path is taken. The old `inlang project validate` command is now relatively ugly. 

```
inlang project validate --project ./lucky-elephant.inlang
```

## Proposal

Remove the project nesting as part of the v2 release.

Question: Is there anything else that needs to be validated in the future? Seems like everything is project related anyways i.e. modules, lint rules, etc. 

```diff
-inlang project validate --project ./lucky-elephant.inlang
+inlang validate --project ./lucky-elephant.inlang
```

**Comment by felixhaeberle** [comment]
this makes perfect sense as inlang is always a "project" now with #1762 👍

**Comment by samuelstroschein** [comment]
implemented in [2cafaf5](https://github.com/inlang/monorepo/commit/2cafaf56c00b1887fbe5badb2e36a09686de334f)
